generated/nidcpower/nidcpower.proto

```diff
@@ -424,8 +424,8 @@ enum NiDCPowerInt32AttributeValues {
   NIDCPOWER_INT32_COMPLIANCE_LIMIT_SYMMETRY_VAL_ASYMMETRIC = 1;
   NIDCPOWER_INT32_CURRENT_LIMIT_BEHAVIOR_VAL_CURRENT_REGULATE = 0;
   NIDCPOWER_INT32_CURRENT_LIMIT_BEHAVIOR_VAL_CURRENT_TRIP = 1;
```

Comment:
@harsha-bhushan @doshirohan 

This is all of the changes to existing enums if we were to just change the behavior.

Any existing code will continue to work **until** you re-run protoc. At that point it will be "source-broken" and the user needs to patch the enums.

ALTERNATIVE:
1. Add an `introduced_version` to the configuration for each driver as needed. Use the version to switch to legacy behavior on codepaths where it's known to break APIs. (Default to the new behavior and use on any codepaths that don't currently change the output).
2. Reject this change and leave it as is.

Reply:
I like the new names better, we should have thought of this before. I think source-broken code is acceptable in this case since this is something that would result in obvious error when you upgrade to newer proto files and won't be something hard to debug/fix.

Reply:
I'll check Cifra's thoughts once & get back. I'm not too concerned about the breaking the source code in this case, since it affects only a few enums & in languages other than python, intellisence would help the user remove & select new name.